R/fitDbmToPeaks.R

```diff
@@ -3,6 +3,17 @@
 ## Copyright (c) Felix Andrews <felix@nfrac.org>
 ##
 
+#' Title
+#'
+#' Placeholder description
+#'
+#' @param MODEL Placeholder
+#' @param objective Placeholder
+#' @param P.thresh Placeholder
+#' @param delay Placeholder
+#' @param return_fit Placeholder
+#'
+#' @export
 fitDbmToPeaks <-
   function(MODEL,
```

Comment:
Was this exported originally?
I might need to check where it is used.

Reply:
Yup, was also exported originally.

[https://github.com/josephguillaume/hydromad/blob/master/NAMESPACE](https://github.com/josephguillaume/hydromad/blob/master/NAMESPACE)

Reply:
To be addressed by #69